frontend/frontend-common/components/skeleton/FilterSkeleton.tsx

```diff
@@ -0,0 +1,13 @@
+import { Skeleton, VStack } from "@navikt/ds-react";
+
+export function FilterSkeleton() {
```

Comment:
Det er jo litt forskjellig markup på de ulike skeletonene vi har brukt enn så lenge. Dvs. at om vi tar inn denne endringen vil det ene filteret se litt "rart" ut når denne komponenten brukes alle steder vi har hatt markup for skeletons.

Reply:
Jeg så på det, men det var bare en liten forskjell i de to Skeletonene. Jeg tenker vi kan bruke denne i alle filtere, så er det likt på de. Filterne er jo forskjellig uansett fra veilederflaten og adminflaten, og jeg tviler på at brukerne bryr seg om det er annerledes Skeleton nå enn før

Reply:
Hva mener du med at det vil se rart ut?